test/extended/dr/backup_restore.go

```diff
@@ -0,0 +1,289 @@
+package dr
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/disruption"
+)
+
+var _ = g.Describe("[sig-etcd][Feature:DisasterRecovery][Disruptive]", func() {
+	defer g.GinkgoRecover()
+
+	f := framework.NewDefaultFramework("backup-restore")
+	f.SkipNamespaceCreation = true
+	f.SkipPrivilegedPSPBinding = true
+
+	oc := exutil.NewCLIWithoutNamespace("backup-restore")
+
+	// Validate the documented backup and restore procedure as closely as possible:
+	//
+	// backup: https://docs.openshift.com/container-platform/4.6/backup_and_restore/backing-up-etcd.html
+	// restore: https://docs.openshift.com/container-platform/4.6/backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html
+	//
+	// Comments like 'Backup 2' and 'Restore '1a' indicate where a test step
+	// corresponds to a step in the documentation.
+	//
+	// Backing up and recovering on the same node is tested by quorum_restore.go.
+	g.It("[Feature:EtcdRecovery] Cluster should recover from a backup taken on one node and recovered on another", func() {
+		masters := masterNodes(oc)
+		// Need one node to backup from and another to restore to
+		o.Expect(len(masters)).To(o.BeNumerically(">=", 2))
+
+		// Pick one node to backup on
+		backupNode := masters[0]
+		framework.Logf("Selecting node %q as the backup host", backupNode.Name)
+
+		// Recovery 1
+		// Pick a different node to recover on
+		recoveryNode := masters[1]
+		framework.Logf("Selecting node %q as the recovery host", recoveryNode.Name)
+
+		// Recovery 2
+		g.By("Verifying that all masters are reachable via ssh")
```

Comment:
I think 4's description is insufficient in the docs - not only must the machines be stopped, but if they are lost they need to be incapable of being turned back on.  It is the administrators responsibility to ensure the machines do not start once step 4 has been completed.  

I would say the docs for 4 should say, instead of :

> 4. Stop the static pods on all other control plane nodes.

be

> 4. Ensure all control plane nodes EXCEPT the recovery host are terminated and cannot be restarted, or have their processes permanently stopped
> Failure to ensure that only the recovery host is running instances of the control plane software will result in data corruption and workload disruption.  You MUST ensure the non-functional hosts cannot start control plane software by following these instructions.

And probably a discussion of why this is the case (we're ensuring that no older quorum can form).

Reply:
We might want to simply break the non-recovery hosts into two classes in the description of 4: those that are known functional, and those that are unrecoverable / known faulty.  The known functional can be stopped (allows us to potentially recover with them if the current host fails).  The known faulty need to be prevented from becoming running again.